Chisel/CHLDuplicateDetection.m

```diff
@@ -0,0 +1,47 @@
+//
+//  CHLDuplicateDetection.m
+//  Chisel
+//
+//  Created by Greg Pstrucha on 3/1/16.
+//  Copyright © 2016 Facebook. All rights reserved.
+//
+
+#if __has_feature(objc_arc)
+#error CHLDuplicateDetection.m expects ARC to be disabled.
```

Comment:
This file doesn't need ARC disabled. And actually, it needs ARC enabled because there are no `-release` calls, so the objects allocated in here are leaking.